helpers.js

```diff
@@ -246,4 +246,9 @@ helpers.findNearestTeamMember = function (gameData) {
     return pathInfoObject.direction;
 };
 
+// Clears the console's screen
+helpers.clearScreen = function() {
```

Comment:
I can see why you put `clearScreen` in this file, but would you please move it to *test-battle.js*? The *hero.js* and *helper.js* files should be completely modifiable by the player, so *test-battle.js* shouldn't depend on them for much more than what move the hero is going to make.